packages/pluggableWidgets/datagrid-number-filter-web/src/DatagridNumberFilter.editorConfig.ts

```diff
@@ -34,32 +34,36 @@ export const getPreview = (values: DatagridNumberFilterPreviewProps): StructureP
                 columnSize: "grow",
                 backgroundColor: "#FFFFFF",
                 children: [
-                    {
-                        type: "Container",
-                        padding: 4,
-                        grow: 0,
-                        children: [
-                            {
-                                type: "Image",
-                                document: getSvgContent(values.defaultFilter)
-                            } as ImageProps
-                        ]
-                    } as ContainerProps,
-                    {
-                        type: "Container",
-                        borders: true,
-                        borderWidth: 0.5,
-                        grow: 0
-                    } as ContainerProps,
+                    ...(values.adjustable
+                        ? [
+                              {
+                                  type: "Container",
+                                  padding: 2,
+                                  grow: 0,
+                                  children: [
+                                      {
+                                          type: "Image",
+                                          document: getSvgContent(values.defaultFilter)
+                                      } as ImageProps
+                                  ]
+                              } as ContainerProps,
+                              {
+                                  type: "Container",
+                                  borders: true,
+                                  borderWidth: 0.5,
+                                  grow: 0
+                              } as ContainerProps
+                          ]
+                        : []),
                     {
                         type: "Container",
                         padding: 8,
                         children: [
                             {
                                 type: "Text",
-                                fontColor: "#BBBBBB",
+                                fontColor: values.placeholder ? "#BBBBBB" : "#FFF",
                                 italic: true,
-                                content: values.placeholder ?? ""
+                                content: values.placeholder.length > 0 ? values.placeholder : "Sample"
```

Comment:
Similar comment here
```suggestion
                                content: values.placeholder ? values.placeholder : "Sample"
```

Reply:
Fixed